Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -110,7 +110,7 @@ const ConfigInfo<bool> GFX_ENHANCE_DISABLE_COPY_FILTER{
 const ConfigInfo<bool> GFX_ENHANCE_ARBITRARY_MIPMAP_DETECTION{
     {System::GFX, "Enhancements", "ArbitraryMipmapDetection"}, true};
 const ConfigInfo<float> GFX_ENHANCE_ARBITRARY_MIPMAP_DETECTION_THRESHOLD{
-    {System::GFX, "Enhancements", "ArbitraryMipmapDetectionThreshold"}, 4.5f};
+    {System::GFX, "Enhancements", "ArbitraryMipmapDetectionThreshold"}, 14.0f};
```

Comment:
14% sounds like a too huge treshhold. Do we really need such a high treshhold?

Reply:
It's not a linear scale anymore, so yes.

Reply:
How is this going to deal with old configs that have already been written and will still keep the previous value?

Reply:
Dolphin doesn't write settings to disk unless they're changed, and this was only added a week or so ago.

Reply:
That's a lot of users getting broken configs. OTOH I just checked my config directory and I don't see the setting in there. Is it just not getting written at all right now? That would be lucky.

Reply:
No, as I said Dolphin doesn't write default values to disk.